lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/Sequential.hs

```diff
@@ -566,27 +701,31 @@ data SeqState (n :: NetworkDiscriminant) k = SeqState
         -- ^ Reward account public key associated with this wallet
     , derivationPrefix :: DerivationPrefix
         -- ^ Derivation path prefix from a root key up to the internal account
+    , scriptPool :: !(VerificationKeyPool k)
```

Comment:
A little comment for the future readers :pray:

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/ab8567640737c089cb9f15c4be3cfce6dbcadb1a